### PR TITLE
Add Netlify Redirects

### PR DESCRIPTION
**Pull request by anabellabuckvar**
## DESCRIPTION
This PR replicates all existing redirects in the `config/redirects` file so that they they bahvae as expected in Netlify staging builds

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by branberry — APPROVED** [review]
They work for me! Thanks for adding these

File: netlify.toml [hunk 1]
```diff
@@ -4,3 +4,28 @@ name = "snooty-cache-plugin"
...
+to = "/getting-started/migration-scenarios"
+
+[[redirects]]
+from = "/supported-databases"
+to = "/getting-started/supported-databases"
+
+[[redirects]]
+from = "/connection-strings/connection-strings"
+to = "/release-notes"
+force = true
```
Comment (ianf-mongodb) [inline netlify.toml:31]: I think this may be left over from our intial testing session, we should remove this one :)

```suggestion

```

**Review by ianf-mongodb — COMMENTED** [review]
Thanks @anabellabuckvar, just one small change here. I tested the other redirects and they work great!

File: netlify.toml (hunk 1 above)
Comment (anabellabuckvar) [reply netlify.toml:31]: Good catch, thank you!

**Pull request merged**